cfme/configure/configuration/region_settings.py

```diff
@@ -368,27 +368,32 @@ def update(self, updates, cancel=False):
                         by defaul map will be updated
         """
         view = navigate_to(self, 'Edit')
-        view.fill(self._form_mapping(**updates))
+        # only category can be updated, as other fields disabled by default
+        view.fill({
+            'category': updates.get('category')
+        })
 
         if cancel:
             view.cancel_button.click()
-            flash_message = 'Edit of Container Label Tag Mapping "fff" was cancelled by the user'
+            flash_message = (
+                'Edit of Container Label Tag Mapping "{}" was cancelled by the user'.format(
+                    self.label)
+            )
         else:
             view.save_button.click()
             flash_message = 'Container Label Tag Mapping "{}" was saved'.format(self.label)
 
         view = self.create_view(MapTagsAllView, override=updates)
         view.flash.assert_success_message(flash_message)
 
-    def delete(self, cancel=True):
+    def delete(self, cancel=False):
```

Comment:
no, it won't affect, there are no existing tests for map tags now. But they are just in progress in https://github.com/ManageIQ/integration_tests/pull/5036 PR